linux/home/.tmux.conf

```diff
@@ -0,0 +1,19 @@
+set -g prefix C-Space
+set-window-option -g mode-keys vi
+
+# no-prefix
+bind -n C-w kill-pane
+bind -n C-t new-window
+bind -n C-1 select-window -t 0
+bind -n C-2 select-window -t 1
+bind -n C-3 select-window -t 2
+bind -n C-4 select-window -t 3
+bind -n C-5 select-window -t 4
+bind -n C-6 select-window -t 5
+bind -n C-7 select-window -t 6
+bind -n C-8 select-window -t 7
+bind -n C-9 select-window -t 8
+
+# prefixed
+bind-key -T prefix b split-window -h
+bind-key -T prefix v split-window -v
```

Comment:
set shell
```
# double check the symlink here, or use `/bin/zsh` instead
set-option -g default-shell /usr/bin/zsh
```

rebind to vim rather than this
```
bind-key -r -T prefix       Up                   select-pane -U
bind-key -r -T prefix       Down                 select-pane -D
bind-key -r -T prefix       Left                 select-pane -L
bind-key -r -T prefix       Right                select-pane -R
```